### ISSUE TITLE
Blender is not affected by this vulnerability

### ISSUE THREAD

**Issue by qzwxsaedc** [YfryTchsGD/Log4jAttackSurface#20]
![image](https://user-images.githubusercontent.com/23263915/145675270-4fadd3cd-5da4-445f-b2a4-8def3fd72388.png)
@cckuailong 别tm瞎提交

**Comment by chinoll** [comment]
fake news!
![image](https://user-images.githubusercontent.com/22575232/145675946-608aca84-110a-4c67-98fc-fa9d3eddc4f1.png)

**Comment by cckuailong** [comment]
> ![image](https://user-images.githubusercontent.com/23263915/145675270-4fadd3cd-5da4-445f-b2a4-8def3fd72388.png)
> @cckuailong 别tm瞎提交

本来就是tm个信息收集的东西，信息不对，干掉就好了，而且承认错误，但是你怨气这么大干啥？？？注意素质

**Comment by Zy143L** [comment]
Liar

**Comment by cckuailong** [comment]
it come from twitter，and i didnot verify it，sorry about the fake info. I’ll test before pr next time.  qwq

**Comment by qzwxsaedc** [comment]
> 

英文里唯唯诺诺，中文里重拳出击是吧。

怼的就是你这种人，提交的数据不检查，还"承认错误"，道歉都在23分钟后才发出来，你咋这么要脸呢。

**Comment by cckuailong** [comment]
> > 
> 
> 英文里唯唯诺诺，中文里重拳出击是吧。
> 
> 怼的就是你这种人，提交的数据不检查，还"承认错误"，道歉都在23分钟后才发出来，你咋这么要脸呢。

确实是我的错，但是不做就不会犯错，谁没犯过错误呢，都是为了开源项目，目的初衷是好的，希望理解。我也做了很多开源，绝不是恶意提交者！再次抱歉！

**Comment by qzwxsaedc** [comment]
> > > 
> > 
> > 
> > 英文里唯唯诺诺，中文里重拳出击是吧。
> > 怼的就是你这种人，提交的数据不检查，还"承认错误"，道歉都在23分钟后才发出来，你咋这么要脸呢。
> 
> 确实是我的错，但是不做就不会犯错，谁没犯过错误呢，都是为了开源项目，目的初衷是好的，希望理解。我也做了很多开源，绝不是恶意提交者！再次抱歉！

之前以为是恶意提交，语气有点冲，还请见谅。能把数据修正过来自然是最好的结果。

**Comment by ArchaeanDragon** [comment]
Yeah, there are a lot of "rich text" -type input fields in a lot of apps out there, which parse and try to resolve URLs (why Blender would be doing this in an object name is beyond me, but it may be just a common data input library feature they use for all or most input fields), so just a DNS lookup hit check isn't sufficient to determine vulnerability. Blender has no Java in it, and thus cannot use log4j2, and is not vulnerable to this CVE.

**Comment by robertguetzkow** [comment]
I missed that there was already a ticket for this and created a duplicate in #42 by accident. The origin of that screenshot is a tweet that was posted as a joke. Blender is not affected by the CVE.

https://twitter.com/chfourchfour/status/1469412054549286928
